source/utilities.tex

```diff
@@ -2920,6 +2920,18 @@
 
 \rSec3[optional.ctor]{Constructors}
 
+\pnum
+The exposition-only variable template \exposid{converts-from-any-cvref}
+is used by some constructors for \tcode{optional}.
+\begin{codeblock}
+template<class T, class W>
+inline constexpr bool @\exposid{converts-from-any-cvref}@  // \expos
+  = disjunction_v<is_constructible<T, W&>, is_constructible<T, W>,
+                 is_constructible<T, const W&>, is_constructible<T, const W>,
+                 is_convertible<W&, T>, is_convertible<W, T>,
+                 is_convertible<const W&, T>, is_convertible<const W, T>>;
```

Comment:
We're spending four lines here. Shouldn't we group the is_constructibe and is_convertible on a single line (for the same pair of arguments, just reversed because reasons)?

Reply:
Done